sirepo/srunit.py

```diff
@@ -621,3 +627,27 @@ def __req(self, route_or_uri, params, query, op, raw_response, **kwargs):
                     pkdexc(),
                 )
             raise
+
+
+class _UwsgiClient(PKDict):
+
+    SR_SIM_TYPE_DEFAULT = MYAPP
+
+    def sr_post(self, route_or_uri, data, headers=None):
```

Comment:
sr_post is required in `_TestClient` because it subclasses flask.testing.FlaskClient, but that's not the case here so no need for the sr_ prefix. I would call it http_post or something like that.

Reply:
The conftest needs to know what kind of client it is working with. Still worth it? https://github.com/radiasoft/sirepo/blob/5ab3d290c5b44d8d8b44c4a24f392cf422a32f66/tests/conftest.py#L370

Reply:
Oh, I see. Good design. :)